tests/Functional/Action/WebHook/UpdateLineItemsWebhookActionTest.php

```diff
@@ -82,7 +113,10 @@ public function testItThrowsBadRequestIfRequestIsInvalid(
             '/api/v1/web-hooks/update-line-items',
             [],
             [],
-            [],
+            [
+                'PHP_AUTH_USER' => 'testUsername',
+                'PHP_AUTH_PW' => 'testPassword'
```

Comment:
Since you have those credentials declared in `.env.test` - maybe it would be more useful to use it not as the "plaintext input" but grabbed from env params? Or am I missing smth?

Reply:
It would be possible to grab it, but I don't think make much sense since it's not what an user would do, they should only match.